redash/query_runner/mongodb.py

```diff
@@ -266,8 +266,8 @@ def run_query(self, query, user):
 
         try:
             query_data = parse_query_json(query)
-        except ValueError:
-            return None, "Invalid query format. The query is not a valid JSON."
+        except ValueError as error:
+            return None, error.__str__()
```

Comment:
We might want to keep the old message and just append the error message that is returned.